### ISSUE TITLE
Create new UCM collection: UC Merced Library UCCE Santa Barbara and San Luis Obispo Counties

### ISSUE THREAD

**Issue by elopatin-uc3** [CDLUC3/mrt-doc#700]

**Comment by elopatin-uc3** [comment]
Collection intake form created:
https://docs.google.com/document/d/1viJq7ABbviUWaiz834qZ8Dcr1lrpg9feYilhN_oOK1M/edit?usp=sharing

ARK: ark:/13030/m53v5q5r
https://ezid.cdlib.org/id/ark:/13030/m53v5q5r

**Comment by elopatin-uc3** [comment]
Registry entry now available for this collection:
https://registry.cdlib.org/27428/santa-barbara-ans-san-luis-obispo-counties-uc-coop/

@mreyescdl
The associated feed should available on Tuesday 7/20:
https://s3.amazonaws.com/static.ucldc.cdlib.org/merritt/ucldc_collection_27428.atom

**Comment by elopatin-uc3** [comment]
PRs for profiles:
Stage: https://github.com/cdlib/mrt-ingest-profile/pull/28
Prod: https://github.com/cdlib/mrt-ingest-profile/pull/29

**Comment by elopatin-uc3** [comment]
@mreyescdl this ATOM feed is available as well. Looks like this will be a much bigger collection vs. UCSF WAN.

**Comment by mreyescdl** [comment]
Stage admin collection object created
Stage Nuxeo script created and currently running (1674 objects)

@elopatin-uc3 For production, should we alter the notification email to only be internal during the large submission process, or is it okay to send out to UCM?

**Comment by elopatin-uc3** [comment]
@mreyescdl Yes, I will remove the notification email addresses from the production profile. For future updates, where it's likely that fewer objects will generate updates in the feed than the initial harvest, should we plan to put them back in? I'm on the fence, as every once and a while, it seems like there's potential for the entire collection to be updated (based on what you mentioned in the past).

**Comment by elopatin-uc3** [comment]
End user notifications have now been removed from the production profile @mreyescdl .

**Comment by mreyescdl** [comment]
I'm for putting them back in after the submission, but letting the end-user know that systematic metadata updates by them may trigger a bulk resubmission. 
However, the idea of spamming a list-serve is a bit frightening.  Let's discuss on Friday with the group ?

**Comment by elopatin-uc3** [comment]
Sounds good.

**Comment by mreyescdl** [comment]
Stage submission is complete, 1674 objects in total

Please have user review: https://merritt-stage.cdlib.org/m/ucm_lib_ucce_sbslo

**Comment by mreyescdl** [comment]
- Production object created
- Production Nuxeo script created and Puppetized, but not yet enabled

TO DO:  Run Nuxeo script manually and enable cron after Stage approval

**Comment by elopatin-uc3** [comment]
@mreyescdl I received a note from the archivist at UCM this morning that they were going to meet as a group and review content on Stage. If/when they confirm all looks OK, I'll let you know the coast is clear to run the Nuxeo script and enable the cron.

**Comment by elopatin-uc3** [comment]
@mreyescdl the group at UCM followed up and they are ready to have this collection move to production. Could you manually run the Nuxeo script next Wednesday or Thursday (9/22 or 23)?

**Comment by mreyescdl** [comment]
Manually running on production host.  Submitted Wednesday and should complete late Thursday.

Cron is enabled to run weekly on Thursday 06:30

**Comment by elopatin-uc3** [comment]
Thanks @mreyescdl !

**Comment by mreyescdl** [comment]
Production submission is complete, 1674 objects in total

**Comment by elopatin-uc3** [comment]
Thanks Mark. Going to move this to Done.
